src/main/kotlin/se/uulm/snowballr/backend/repository/PaperTableRepo.kt

```diff
@@ -0,0 +1,52 @@
+package se.uulm.snowballr.backend.repository
+
+import org.jetbrains.exposed.sql.ResultRow
+import org.jetbrains.exposed.sql.selectAll
+import se.uulm.snowballr.backend.db.IDatabase
+import se.uulm.snowballr.backend.model.EntityType
+import se.uulm.snowballr.backend.model.SnowballRException.NotFoundException
+import se.uulm.snowballr.backend.model.dto.Paper
+import se.uulm.snowballr.backend.table.PaperTable
+import se.uulm.snowballr.backend.table.toPaper
+import java.util.UUID
+
+/**
+ * Defines an interface for repository operations related to the [PaperTable].
+ *
+ * This interface provides abstraction for handling persistence and retrieval operations for papers. By using this
+ * interface, the functionality for managing papers can remain decoupled from the specifics of the database layer.
+ */
+interface IPaperTableRepo {
+    /**
+     * Returns a paper by its ID or throws a [NotFoundException] if the paper with the passed [id] doesn't exist.
+     */
+    suspend fun getPaperById(id: UUID): Paper
+
+    /**
+     * @returns whether the paper with the passed [id] exists.
```

Comment:
`@returns` doesn't exist in KDoc. There's another occurrence in `FetcherManager`.
```suggestion
     * @return Whether the paper with the passed [id] exists.
```

Reply:
Removed all occurrences I could find :+1: